frontend/src/types/analytics.ts

```diff
@@ -8,6 +8,7 @@ import type { FilterCategory } from "~/constants/filters"
 import { ResultKind } from "~/types/result"
 
 import { RequestKind } from "./fetch-state"
+import { Collection } from "./search"
```

Comment:
This and the previous imports use relative path, while we usually use absolute. I'd prefer to use absolute paths everywhere (and add `import type` if it's a type)

Reply:
Hmmm, it feels weird that ESLint didn't catch and report this. We should look into that, but separately from this PR.

Reply:
Ah, nice catch! These were auto imports from vscode so I didn't even check them. I'll fix